cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -854,21 +864,35 @@
       ]
     },
     "UseMasterSubnetForCompute": {
-      "Fn::And": [
+      "Fn::Or": [
```

Comment:
Where is this diff coming from?

Reply:
This was from some other commit that I took part of my code from. I restored the original code.